autoload/SpaceVim/layers/tmux.vim

```diff
@@ -24,8 +27,14 @@ endfunction
 
 function! SpaceVim#layers#tmux#config() abort
     let g:tmux_navigator_no_mappings = 1
-    nnoremap <silent> <C-h> :TmuxNavigateLeft<cr>
-    nnoremap <silent> <C-j> :TmuxNavigateDown<cr>
-    nnoremap <silent> <C-k> :TmuxNavigateUp<cr>
-    nnoremap <silent> <C-l> :TmuxNavigateRight<cr>
+    augroup custom_config
```

Comment:
Changed the name in commit 707c8e4. Also changed the name of the script function just for good measure.